.travis.yml

```diff
@@ -3,7 +3,7 @@ os: linux
 dist: focal
 
 vm:
-  size: large
+  size: x-large
```

Comment:
Do we want to keep the `x-large` here? This surely means that we won't be able to run as many simulations in parallel?

Reply:
Furthermore, I'm not sure that this is taken into account by travis since the logs still indicate `nproc = 4`.
Maybe it's because the support imposes `large` by default? 
I remember I tried to get one build with `medium` VM and I could not shrink the VM either.
Something that should be discussed with support I think.

Reply:
it is not taken into account. I will change it back.